Rough_Semester_Plan_DS1

```diff
@@ -0,0 +1,17 @@
+16 weeks in the semester, 13 to end before Thanksgiving
```

Comment:
go ahead and modify the syllabus md file, and then delete this file. i'll also be adding a bunch of the Mines boiler plate to the syllabus too...